Sources/Build/BuildDescription/SwiftModuleBuildDescription.swift

```diff
@@ -264,6 +268,18 @@ public final class SwiftModuleBuildDescription {
     /// Whether to disable sandboxing (e.g. for macros).
     private let shouldDisableSandbox: Bool
 
+    /// For Windows we default to static objects and create copies for objects
+    /// That export symbols. This will allow consumers to select which one they want.
+    public enum WindowsTargetType {
+        case `static`
+        case exporting
```

Comment:
Can we use `dynamic` instead?

Reply:
:). I had been using `dynamic` until the very last minute. It's really about exporting symbols from DLLs and was starting to think of this in C terms as in static symbols versus `__declspec(dllexport)`. Maybe dllexport is a better name?

Reply:
DLL is **Dynamic** Link Library, I think the "dynamic" terminology might be clearer / more familiar / approachable for folks compared to something that might be more _technically_ accurate.

Reply:
Fair. I'm only voting 1/2 since it literally was "dynamic" until a hour before I did this push :). I'll change it when I add some tests in the next push.

Reply:
Done.